steps.go

```diff
@@ -433,22 +433,23 @@ func (apiCtx *APIContext) IGenerateARandomIntInTheRangeToAndSaveItAs(from, to in
 
 // IGenerateARandomFloatInTheRangeToAndSaveItAs generates random float from provided range
 // and preserve it under given cacheKey key.
-func (apiCtx *APIContext) IGenerateARandomFloatInTheRangeToAndSaveItAs(from, to int, cacheKey string) error {
-	randInt, err := mathutils.RandomInt(from, to)
-	if err != nil {
-		return fmt.Errorf("problem during generating pseudo random float, randomInt err: %w", err)
-	}
-
-	float01 := rand.Float64()
-	strFloat := fmt.Sprintf("%.2f", float01*float64(randInt))
-	floatVal, err := strconv.ParseFloat(strFloat, 64)
-	if err != nil {
-		return fmt.Errorf("problem during generating pseudo random float, parsing err: %w", err)
+func (apiCtx *APIContext) IGenerateARandomFloatInTheRangeToAndSaveItAs(from, to float64, cacheKey string) func(from, to int, cacheKey string) error {
+	return func(from, to float64, cacheKey string) error {
```

Comment:
Hi, method signature states, that return value will be `func(from, to int, cacheKey string) error`.
Line 437 returns slightly different function `return func(from, to float64, cacheKey string) error`. This is subject to change. 

According to discussion in #14 , please try to create slightly different method, similar to:
```go
// GenerateFloat64 creates pseudo random float64 generator with predefined rounding precision
func (apiCtx *APIContext) CreateFloat64Generator(round int) func(from, to float64, cacheKey string) error
```

If you need any help, ask anything